pkg/cmd/testserver/testserver.go

```diff
@@ -96,17 +100,51 @@ func (c *Config) Complete() (RunnableTestServer, error) {
 		return nil, err
 	}
 
+	gatewayHandler, err := gateway.NewHandler(context.TODO(), c.GRPCServer.Address, c.GRPCServer.TLSCertPath)
```

Comment:
`context.Background()` unless we want to leave `TODO` here to signal that at some point we need to thread a context into this function

Reply:
Ah; I copied this directly from the existing serve implementation. @ecordell any idea why we have a `TODO` there as well?

Reply:
And I also was just matching what was there [originally](https://github.com/jzelinskie/spicedb/blob/e9b164af2de444cfc945b752b27fc0fbbcb36d06/cmd/spicedb/serve.go#L388) before I moved initialization into the cmd package 😄 

I think it would need to get refactored a good bit, ideally nothing that required a context would happen until you call `Run(ctx)`